src/component/mxgraph/GraphCellUpdater.ts

```diff
@@ -108,12 +109,31 @@ export default class GraphCellUpdater {
           font.isStrikeThrough !== undefined &&
             (cellStyle = mxgraph.mxUtils.setStyleFlag(cellStyle, mxgraph.mxConstants.STYLE_FONTSTYLE, mxgraph.mxConstants.FONT_STRIKETHROUGH, font.isStrikeThrough));
 
-          // TODO To uncomment when we implement the Opacity in global/background/font/stroke
+          // TODO To uncomment when we implement the Opacity in global/fill/font/stroke
           // font.opacity && (cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_TEXT_OPACITY, font.opacity));
         }
 
+        if (isShapeStyleUpdate(styleUpdate)) {
```

Comment:
**issue:** SonarCloud complains about this method

![image](https://user-images.githubusercontent.com/27200110/225386915-ed022197-4e20-4dc4-a4b4-648cd184661a.png)